node-operators/networks/collator.md

```diff
@@ -142,12 +156,15 @@ Make sure you write down this public key of the author ID. Next, this will be ma
 
 Once you've generated your author ID (session keys), the next step is to map it to your H160 account (an Ethereum-styled address). Make sure you hold the private keys to this account, as this is where the block rewards are paid out to.
 
-There is a {{ networks.moonbase.staking.collator_map_bond }} DEV tokens bond that is sent when mapping your authord ID with your account. This bond is per author ID registered.
+There is a bond that is sent when mapping your author ID with your account. This bond is per author ID registered. The bond set is as follows:
+
+ - Moonbase Alpha - {{ networks.moonbase.staking.collator_map_bond }} DEV tokens 
+ - Moonriver - {{ networks.moonriver.staking.collator_map_bond }} MOVR tokens. 
 
 The `authorMapping` module has the following extrinsics programmed:
 
- - **addAssociation**(*address* authorID) — maps your author ID to the H160 account from which the transaction is being sent, ensuring is the true owner of its private keys. It requires a {{ networks.moonbase.staking.collator_map_bond }} DEV tokens bond
- - **clearAssociation**(*address* authorID) — clears the association of an author ID to the H160 account from which the transaction is being sent, which needs to be the owner of that author ID. Also refunds the {{ networks.moonbase.staking.collator_map_bond }} DEV tokens bond
+ - **addAssociation**(*address* authorID) — maps your author ID to the H160 account from which the transaction is being sent, ensuring is the true owner of its private keys. It requires a bond
```

Comment:
maybe add a hyperlink to where to find it?

Reply:
a hyperlink to find what?

Reply:
You removed the bond information, you can add a link to where the table is with the bond information, like 'it requires a [bond](#link-to-section)`